README.md

```diff
@@ -256,6 +256,8 @@ The `OTNetworkTest()` constructor includes the following parameters:
     (`true`) or not (`false`, the default). Disabling scalable video
     was added in OpenTok.js version 2.24.7.
 
+  * `fullHd` (Boolean) -- (Optional) Allows publishing with a resolution of 1080x1920. If the camera does not support 1080x1920 resolution, an `UNSUPPORTED_RESOLUTION_ERROR` will be returned.
```

Comment:
The `UNSUPPORTED_RESOLUTION_ERROR` is not _returned_. The `OTNetworkTest.testConnectivity()` method is rejected with an error that has the name property set to `UNSUPPORTED_RESOLUTION_ERROR`, right?